android/src/main/kotlin/com/phyllo/connect/phyllo_connect/PhylloConnectPlugin.kt

```diff
@@ -48,15 +48,42 @@ class PhylloConnectPlugin : FlutterPlugin, MethodCallHandler, EventChannel.Strea
             }
             "initialize" -> {
 
-                Log.d(logTag, call.arguments.toString())
+                val clientDisplayName = call.argument<String?>("clientDisplayName")
+                val userId = call.argument<String?>("userId")
+                val token = call.argument<String?>("token")
+                val environment = call.argument<String?>("environment")
+                val workPlatformId = call.argument<String?>("workPlatformId") ?: ""
+                val singleAccount = call.argument<Boolean?>("singleAccount") ?: false
```

Comment:
This shouldn't be exposed

Reply:
it's always exposed because we need to pass from flutter sdk , its bridge method with native sdk , we had already discussed I think so .

Reply:
No, since everything is moved to JSON or dictionary structure, it shouldn't be there